frontend/packages/dev-console/src/components/topology/filters/TopologyFilterBar.tsx

```diff
@@ -37,11 +48,30 @@ const TopologyFilterBar: React.FC<TopologyFilterBarProps> = ({
     </ToolbarGroup>
     <ToolbarGroup className="odc-topology-filter-bar__search">
       <ToolbarItem>
-        <TextFilter
-          label="name"
-          value={searchQuery}
-          onChange={(e) => onSearchQueryChange(e.target.value)}
-        />
+        <Split>
+          <SplitItem>
+            <TextFilter
+              placeholder="Find by name..."
+              value={searchQuery}
+              onChange={(e) => onSearchQueryChange(e.target.value)}
+            />
+          </SplitItem>
+          <SplitItem>
+            <Bullseye>
+              <Tooltip
+                position={TooltipPosition.left}
```

Comment:
The tooltip is covering the text-filter if left position is used. It makes sense to confirm this with UX if this is fine. According to me the position should be bottom so that it doesnt cover the text-filter.

cc: @serenamarie125

Reply:
As per the UX design tooltip is on the left.